packages/acceptance-tests/pkg-tests-specs/sources/commands/workspaces/focus.test.js

```diff
@@ -138,19 +138,49 @@ describe(`Commands`, () => {
         }
       )
     );
+
+    test(
+      `should not execute postinstall scripts of unspecified workspace`,
+      makeTemporaryEnv(
+        {
+          private: true,
+          workspaces: [`packages/*`],
+        },
+        async ({path, run}) => {
+          await setupProject(path);
+
+          await run(`install`);
+
+          await xfs.removePromise(ppath.join(path, `.yarn/cache`));
+          await xfs.removePromise(ppath.join(path, `.yarn/build-state.yml`));
+
+          const fooPostinstallPath = ppath.join(path, `packages/foo/postinstall.log`);
+          const quxPostinstallPath = ppath.join(path, `packages/qux/postinstall.log`);
+          await xfs.removePromise(fooPostinstallPath);
+          await xfs.removePromise(quxPostinstallPath);
```

Comment:
Are those lines necessary? Can't we just run the focus as first command?

Reply:
:+1: I'll remove them